components/breadcrumbs/src/item/breadcrumbs-item.spec.ts

```diff
@@ -298,7 +298,7 @@ class TestBreadcrumbsItem {
   external = false;
 
   // tslint:disable-next-line:deprecation
```

Comment:
Do we still need the depraction disable?

Reply:
I think the deprecation of the breadcrumb should be handled in a separate PR:

```
/**
 * @deprecated Use {@link DtBreadcrumbsItem2} instead.
 * @breaking-change To be removed with 6.0.0
 */
```

Reply:
yes, you are right. I was just not sure if its not already resolved. Thank you for clarifying.